src/test/java/org/broadinstitute/hellbender/engine/ReadWalkerGCSSupportIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.PrintReads;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Tests to prove that we can access and query inputs on Google Cloud Storage (GCS) in ReadWalkers.
+ */
+public class ReadWalkerGCSSupportIntegrationTest extends CommandLineProgramTest {
+
+    private static final String TEST_BAM_ON_GCS = "org/broadinstitute/hellbender/engine/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.10000000-10000020.with.unmapped.bam";
```

Comment:
I wanted these basic tests to be fast, so I used a small file containing representative snippets of reads. But we can expand these tests over time -- eg., if you want to add a separate test case to `ReadWalkerGCSSupportIntegrationTest` that uses a larger file to fully exercise the prefetcher, be my guest!